deps/obs-scripting/obslua/obslua.i

```diff
@@ -40,7 +40,7 @@
 %inline %{
 static inline void wrap_blog(int log_level, const char *message)
 {
-        blog(log_level, "%s", message);
+				blog(log_level, "%s", message);
```

Comment:
This is just a whitespace change.  Did you do this, or did clang-format?

Reply:
All of these are due to clang format, let me try revert these